docs/resources/kafka_user.md

```diff
@@ -5,23 +5,42 @@ description: |-
   
 ---
 
-# Resource `instaclustr_kafka_user`
-
-
-
-
-
-## Schema
-
-### Required
-
-- **cluster_id** (String)
-- **password** (String, Sensitive)
-- **username** (String)
-
-### Optional
-
-- **id** (String) The ID of this resource.
-- **initial_permissions** (String)
-
-
+# Resource  `instaclustr_kafka_user` and Data Source `instaclustr_kafka_user_list`                             
```

Comment:
Can we separate the Data Source in another hierarchy (docs/data-sources)?
https://www.terraform.io/docs/registry/providers/docs.html#directory-structure

Reply:
Done, though I'm not entirely sure how this will return to make an example?